src/components/Gesture.svelte

```diff
@@ -270,45 +283,77 @@
-    <div class="h-full flex items-center gap-x-3 p-2">
-      <div class="w-33 flex justify-center items-center gap-x-3">
-        <IconButton
-          ariaLabel={$t(
-            $chosenGesture === gesture
-              ? 'content.data.recordAction'
-              : 'content.data.selectAction',
-            {
-              values: {
-                action: $nameBind,
+{#if showAddActionWalkThrough}
+  <div
+    class="h-full flex w-50 flex-col relative items-center"
+    style="transform: translate(-50px, 50px)">
+    <img
+      class="mb-3 w-30"
+      alt={$t('content.data.addActionWalkThrough.imageAlt')}
+      src={greetingEmojiWithArrowImage} />
+    <p class="text-center">
+      {$t('content.data.addActionWalkThrough')}
+    </p>
+  </div>
+{:else}
+  <div class="max-w-max {isGestureNamed || hasRecordings ? 'visible' : 'invisible'}">
+    <GestureTilePart small elevated>
+      <div class="h-full flex items-center gap-x-3 p-2">
+        <div class="w-33 flex justify-center items-center gap-x-3">
+          <IconButton
+            ariaLabel={$t(
+              $chosenGesture === gesture
+                ? 'content.data.recordAction'
+                : 'content.data.selectAction',
+              {
+                values: {
+                  action: $nameBind,
+                },
               },
-            },
-          )}
-          onClick={$chosenGesture === gesture ? countdownStart : selectClicked}
-          disabled={!$state.isInputConnected}
-          rounded>
-          <RecordIcon
-            class="h-20 w-20 {$chosenGesture === gesture
-              ? 'text-rose-600'
-              : 'text-neutral-400'} flex justify-center items-center rounded-full" />
-        </IconButton>
+            )}
+            onClick={$chosenGesture === gesture ? countdownStart : selectClicked}
+            disabled={!$state.isInputConnected}
+            rounded>
+            <RecordIcon
+              class="h-20 w-20 {$chosenGesture === gesture
+                ? 'text-rose-600'
+                : 'text-neutral-400'} flex justify-center items-center rounded-full" />
+          </IconButton>
+        </div>
+        {#if hasRecordings}
+          {#each $gesture.recordings as recording (String($gesture.ID) + String(recording.ID))}
+            <Recording {recording} onDelete={deleteRecording} />
+          {/each}
+        {/if}
       </div>
-      {#if $gesture.recordings.length > 0}
-        {#each $gesture.recordings as recording (String($gesture.ID) + String(recording.ID))}
-          <Recording {recording} onDelete={deleteRecording} />
-        {/each}
-      {/if}
-    </div>
-  </GestureTilePart>
-</div>
+    </GestureTilePart>
+  </div>
+{/if}
+
+{#if isGestureNamed && showWalkThrough && !hasRecordings && !showCountdown && !isThisRecording}
+  <div></div>
+  <div class="h-full flex" style="transform: translateX(65px)">
+    <img
+      class="w-15"
+      alt={$t('content.data.addRecordingWalkThrough.imageAlt')}
```

Comment:
I think another `alt=""` here as we have descriptive text below. It could be something like "arrow pointing to record button", but that's doesn't help anyone using a screen reader.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/103/commits/6f155e8cca85f2a96f5b76909a8008eb40747645